src/Maestro/tests/Maestro.ScenarioTests/MaestroScenarioTestBase.cs

```diff
@@ -12,9 +12,10 @@
 
 namespace Maestro.ScenarioTests
 {
+    [SetUpFixture]
     public class MaestroScenarioTestBase
     {
-        private TestParameters _parameters;
+        internal TestParameters _parameters;
```

Comment:
This should probably be "protected", and once a field is protected, the naming rules, change, so it should become `Parameters`.